### PR TITLE
AlwaysUseMSOAuthForAutoDiscover reg key missing.

### PR DESCRIPTION
**Pull request by j0rt3g4**
Issue #576 Added key
Copy review @nenonix @JohanFreelancer9 is required on this one guys. Thank you in advance!

### CODE REVIEW

**Comment by j0rt3g4** [comment]
Issue #576

File: Enterprise/modern-auth-for-office-2013-and-2016.md [hunk 1]
```diff
@@ -68,6 +68,7 @@ The following table describes the authentication behavior for Office 2013 or Off
   
 |****Office client app version****|****Registry key present?****|****Modern authentication on?****|****Authentication behavior with modern authentication turned on for the tenant (default)****|****Authentication behavior with modern authentication turned off for the tenant****|
 |:-----|:-----|:-----|:-----|:-----|
+|Office 2016  <br/> |Yes, <br> AlwaysUseMSOAuthForAutoDiscover = 1 <br/> |Yes  <br/> |Force moderm authentication on Outlook 2010,2013 or 2016 <br/> |Forces Moderm authentication within the outlook client.<br/> |
```
Comment (JohanFreelancer9) [inline Enterprise/modern-auth-for-office-2013-and-2016.md]:
```suggestion
|Office 2016  <br/> |Yes, <br> AlwaysUseMSOAuthForAutoDiscover = 1 <br/> |Yes  <br/> |Forces modern authentication on Outlook 2010, 2013 or 2016 <br/> |Forces modern authentication within the Outlook client.<br/> |
```

**Comment by afield1235** [comment]
Note: now that 72+ hours have gone by, new clients are properly authenticating with Modern auth without the key. These new clients are also Azure AD domain joined so that may be a factor. Nonetheless, this key is what solved our initial issue for 2016 clients who weren’t getting the modern auth prompt. 

The referenced KB article doesn’t list this particular scenarios either: https://support.microsoft.com/en-us/help/3126599/outlook-prompts-for-password-when-modern-authentication-is-enabled

**Are these articles not in Github? I can’t submit feedback on it.**

File: Enterprise/modern-auth-for-office-2013-and-2016.md [hunk 2]
```diff
@@ -68,6 +68,7 @@ The following table describes the authentication behavior for Office 2013 or Off
   
 |****Office client app version****|****Registry key present?****|****Modern authentication on?****|****Authentication behavior with modern authentication turned on for the tenant (default)****|****Authentication behavior with modern authentication turned off for the tenant****|
 |:-----|:-----|:-----|:-----|:-----|
+|Office 2016  <br/> |No, <br> AlwaysUseMSOAuthForAutoDiscover = 1 <br/> |Yes  <br/> |Force moderm authentication on Outlook 2010,2013 or 2016 <br/> [More info](https://support.microsoft.com/help/3126599/outlook-prompts-for-password-when-modern-authentication-is-enabled)|Forces Moderm authentication within the outlook client.<br/> |
```
Comment (JohanFreelancer9) [inline Enterprise/modern-auth-for-office-2013-and-2016.md]:
```suggestion
|Office 2016  <br/> |No, <br> AlwaysUseMSOAuthForAutoDiscover = 1 <br/> |Yes  <br/> |Forces modern authentication on Outlook 2010, 2013 or 2016 <br/> [More info](https://support.microsoft.com/help/3126599/outlook-prompts-for-password-when-modern-authentication-is-enabled)|Forces modern authentication within the outlook client.<br/> |
```

**Comment by mypil** [comment]
@MSFTTracyP - PR has been copy edited and is ready for final review  based on issue #576  

Thank you.

**Comment by mypil** [comment]
@MSFTTracyP - We would like to follow up. Can you please review, approve and merge this PR?

@dansimp - Can you help to review and merge this PR?

Thank you.

cc: @kenwith

**Pull request merged**